src/okteto.ts

```diff
@@ -11,6 +11,7 @@ import * as os from 'os';
 import * as download from 'download';
 import * as semver from 'semver';
 import {pascalCase} from 'change-case';
+import *  as paths from './paths';
```

Comment:
nit: remove whitespace

```suggestion
import * as paths from './paths';
```